panacea/query_client_test.go

```diff
@@ -1,17 +1,17 @@
-package panacea_test
+package panacea
```

Comment:
Is the package of this test `panacea`? not `panacea_test`?

Reply:
This should be `panacea` because some tests access private members in the `QueryClient` (actually, I made those members private in this PR).
If we really wanna test this package as an external user (aka. blackbox tests), we should use the `panacea_test` package. But for now, I think it's simpler to use `panacea`.

Reply:
I got it. Thank you!